util/generate-unicode-ranges.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+# Generates the code for a sorted unicode range array as used in hb-ot-os2-unicode-ranges.hh
+# Input is a tab seperated list of unicode ranges from the otspec.
```

Comment:
Link to spec please.

Reply:
util/ is not for these things. Should go in src/ and named 'gen-....py'

Reply:
done